src/applications/personalization/view-dependents/actions/index.js

```diff
@@ -8,11 +9,16 @@ export function fetchAllDependents() {
     const response = await getData('/dependents_applications/show');
 
     if (response.errors) {
+      recordEvent({
+        event: `disability-view-dependents-load-failed`,
+        'error-key': `${response.errors[0].status}_internal_error`,
```

Comment:
Are the underscores necessary here? Looks like this ticket suggests an error-key without them - https://github.com/department-of-veterans-affairs/va.gov-team/issues/6830

Reply:
Jon, the analytics guy, said he would rather have the errors with underscores in Slack https://dsva.slack.com/archives/GV9H4FZGB/p1587667608005200